src/executePrettier.test.js

```diff
@@ -77,6 +77,25 @@ describe('executePrettierOnBufferRange()', () => {
     });
   });
 
+  describe('when file is already pretty', () => {
+    beforeEach(() => {
+      prettier.format.mockImplementation(() => 'untransformed text');
+      prettierEslint.mockImplementation(() => 'untransformed text');
```

Comment:
Can we remove this line? It should only be using `prettierEslint` if you mocked out `helpers.shouldUseEslint.mockImplementation(() => true);`?